keyboards/ymdk/ymd09/rev2/rev2.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include "quantum.h"
+
+#define LAYOUT_ortho_3x3( \
+    K00, K01, K02, \
+    K10, K11, K12, \
+    K20, K21, K22  \
+) { \
+    { K00,   K01,   K02 }, \
+    { K10,   K11,   K12 }, \
+    { K20,   K21,   K22 }  \
+}
```

Comment:
For backwards compatibility, 

```suggestion
}

#define LAYOUT LAYOUT_ortho_3x3
```